packages/grafana-prometheus/src/querycache/QueryCache.ts

```diff
@@ -120,132 +72,9 @@ export class QueryCache<T extends SupportedQueryTypes> {
-
-                      this.pendingAccumulatedEvents.set(value.identity, {
-                        datasource: value.datasource ?? 'N/A',
-                        requestCount: (previous?.requestCount ?? 0) + 1,
-                        savedBytesTotal: (previous?.savedBytesTotal ?? 0) + savedBytes,
-                        initialRequestSize: value.bytes,
-                        lastRequestSize: requestTransferSize,
-                        panelId: currentRequest.panelId?.toString() ?? '',
-                        dashId: currentRequest.dashboardUID ?? '',
-                        expr: currentRequest.expr ?? '',
-                        refreshIntervalMs: currentRequest.refreshIntervalMs ?? 0,
-                        sent: false,
-                        from: currentRequest.from ?? '',
-                        queryRangeSeconds: currentRequest.queryRangeSeconds ?? 0,
-                      });
-
-                      // We don't need to save each subsequent request, only the first one
-                      this.pendingRequestIdsToTargSigs.delete(requestId);
-
-                      return;
-                    }
-                  }
-
-                  // If we didn't return above, this should be the first request, let's save the observed size
-                  this.pendingRequestIdsToTargSigs.set(requestId, { ...currentRequest, bytes: requestTransferSize });
-                }
-              }
-            }
-          }
-        });
-      });
-
-      this.perfObeserver.observe({ type: 'resource', buffered: false });
-
-      setInterval(this.sendPendingTrackingEvents, this.sendEventsInterval);
-
-      // Send any pending profile information when the user navigates away
-      window.addEventListener('beforeunload', this.sendPendingTrackingEvents);
-    }
-  }
-
-  sendPendingTrackingEvents = () => {
-    const entries = this.pendingAccumulatedEvents.entries();
-
-    for (let [key, value] of entries) {
-      if (!value.sent) {
-        const event = {
-          datasource: value.datasource.toString(),
-          requestCount: value.requestCount.toString(),
-          savedBytesTotal: value.savedBytesTotal.toString(),
-          initialRequestSize: value.initialRequestSize.toString(),
-          lastRequestSize: value.lastRequestSize.toString(),
-          panelId: value.panelId.toString(),
-          dashId: value.dashId.toString(),
-          expr: value.expr.toString(),
-          refreshIntervalMs: value.refreshIntervalMs.toString(),
-          from: value.from.toString(),
-          queryRangeSeconds: value.queryRangeSeconds.toString(),
-        };
-
-        if (config.featureToggles.prometheusIncrementalQueryInstrumentation) {
-          reportInteraction('grafana_incremental_queries_profile', event);
-        } else if (faro.api.pushEvent) {
-          faro.api.pushEvent('incremental query response size', event, 'no-interaction', {
-            skipDedupe: true,
-          });
-        }
```

Comment:
@gtk-grafana I think we should keep this part at least. What do you think? Or is it also redundant, now?

Reply:
Nah remove it